ethereumj-core/src/test/resources/genesis/beacon-genesis-parse-test.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I'd add smth with 0's omit

Reply:
without `0x` prefix?

Reply:
No, I mean odd hex symbols for example. To test it's passing through hex helper you've added correctly